frontend/views/containers/dashboard/GroupMembers.vue

```diff
@@ -17,15 +17,17 @@
       :class='invitedBy && "is-pending"'
       :key='username'
     )
-      avatar(v-if='invitedBy' size='sm')
-      avatar-user(v-else :username='username' size='sm')
+      profile-card(:username='username' :isSelf='username === ourUsername')
```

Comment:
I think @sandrina-p mentioned (and I agreed) that this `isSelf` property should be removed. Instead the profile card should have a computed property to do this.